<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>islr</class>
 <widget class="QMainWindow" name="islr">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tabla retención impuesto sobre la renta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tabla de impuesto sobre la renta, se debe actualizar anualmente para poder procesar correctamente las nóminas quincenales. Esta información se usa para poder hacer retenciones de ISLR a los trabajadores cuando así se amerite.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="title">
       <string>Tabla de retención de ISLR:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="tasa3">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="maximum">
            <double>99.989999999999995</double>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="tasa2">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="maximum">
            <double>99.989999999999995</double>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDoubleSpinBox" name="rango1">
           <property name="suffix">
            <string>DOP</string>
           </property>
           <property name="maximum">
            <double>999999999999.989990234375000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Rango 3</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Tasa 3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Rango 2</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QDoubleSpinBox" name="tasa1">
           <property name="suffix">
            <string>%</string>
           </property>
           <property name="maximum">
            <double>99.989999999999995</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Tasa 1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tasa 2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="rango2">
           <property name="suffix">
            <string>DOP</string>
           </property>
           <property name="maximum">
            <double>999999999999.989990234375000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDoubleSpinBox" name="rango3">
           <property name="suffix">
            <string>DOP</string>
           </property>
           <property name="maximum">
            <double>999999999999.989990234375000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Rango 1</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Valor 2</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Valor 1</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QDoubleSpinBox" name="valor1">
           <property name="suffix">
            <string>DOP</string>
           </property>
           <property name="maximum">
            <double>999999999999.989990234375000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="valor2">
           <property name="suffix">
            <string>DOP</string>
           </property>
           <property name="maximum">
            <double>999999999999.989990234375000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Nota:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="nota">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Agregar información del ISLR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>63</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="actualizar">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Al presionar este botón se alctualiza la información suministrada en l abase de datos.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Actualizar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>62</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelar">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cierra la ventana y se pierden las modicifaciones.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>63</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>rango1</tabstop>
  <tabstop>rango2</tabstop>
  <tabstop>rango3</tabstop>
  <tabstop>tasa1</tabstop>
  <tabstop>tasa2</tabstop>
  <tabstop>tasa3</tabstop>
  <tabstop>valor1</tabstop>
  <tabstop>valor2</tabstop>
  <tabstop>nota</tabstop>
  <tabstop>actualizar</tabstop>
  <tabstop>cancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
